.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Declare files that will always have LF line endings on checkout.
+*.sh text eol=lf
```

Comment:
Can you enlighten me why you need this?
Never have seen that option within one of my projects.

Reply:
I work on windows and so usually my shell scripts get checked out with crlf. This makes it impossible to run them via wsl.